samples/zigbee/light_switch/prj.conf

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+CONFIG_NCS_SAMPLES_DEFAULTS=y
+
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_SERIAL=y
+CONFIG_GPIO=y
+
+# Make sure printk is not printing to the UART console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+CONFIG_HEAP_MEM_POOL_SIZE=2048
+
+CONFIG_ZIGBEE=y
+CONFIG_ZIGBEE_ROLE_END_DEVICE=y
+
+# Enable bonding
+CONFIG_NVS=y
+CONFIG_SETTINGS=y
+
+# Enable DK LED and Buttons library
+CONFIG_DK_LIBRARY=y
+
+# This example requires more workqueue stack
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+# Cryptocell is not supported through CSPRNG driver API: NCSDK-4813
+CONFIG_ENTROPY_CC310=n
```

Comment:
I assume this must be updated to reflect latest changes related to DTS.

Reply:
This will be changed in all zigbee examples.